charts/htp-builder/values.yaml

```diff
@@ -67,7 +67,7 @@ refinery:
     repository: honeycombio/refinery
     pullPolicy: IfNotPresent
     # Overrides the image tag whose default is the chart appVersion.
-    tag: "2.9.7"
+    tag: "3.0.0"
```

Comment:
I think this change to htp-builder means this chart needs a version bump, too.

I don't know if we do this in the same PR or do separate PRs. 🤔

Reply:
We have decided to not touch htp-builder